src/components/map/Map.tsx

```diff
@@ -246,6 +249,14 @@ export const Map = ({
                   iconName={marker.iconName ? marker.iconName : undefined}
                 />
               )}
+
+              {calloutTextEnabled && (
+                <Callout style={styles.callout}>
+                  <RegularText smallest center>
+                    {calloutText}
```

Comment:
what about directly `marker.title` instead of creating the variable `calloutText`?

Reply:
Yes, very logical. I've made this change.
https://github.com/smart-village-solutions/smart-village-app-app/commit/219b63c72de3f54270f773026c7b5fa3318b29c8

Reply:
but now you have added an additional check, which needs !!